cfgov/v1/tests/management/commands/test_update_data_snapshot_values.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import call_command
+from django.test import Client, TestCase
+
+from wagtail.wagtailcore.blocks import StreamValue
+
+from scripts import _atomic_helpers as atomic
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_page
+
+django_client = Client()
+
+class UpdateDataSnapshotValuesTestCase(TestCase):
+        def test_data_snapshot(self):
+            """ Data Snapshot correctly renders fields on a Browse Page"""
+            browse_page = BrowsePage(
+                title='Browse Page',
+                slug='browse',
+            )
+
+            # Adds a AUT market to a browse page
+            browse_page.content = StreamValue(
+                browse_page.content.stream_block,
+                [atomic.data_snapshot],
+                True
+            )
+            publish_page(child=browse_page)
+
+            # Call management command to update values
+            call_command('update_data_snapshot_values', '--snapshot_file=v1/tests/fixtures/data_snapshots.json')
```

Comment:
@richaagarwal this sees good to me, if I understand what you're asking correctly, which is how to include test files to test management commands with (as opposed to test fixture data [like this](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.TransactionTestCase.fixtures)).

The only concern I'd have about this is because you're using a relative path, this might not necessarily work if you're not running the test from the `cfgov` directory. In the past I've done some ugliness [like this](https://github.com/cfpb/owning-a-home-api/blob/master/ratechecker/tests/data/test_data.py#L10) that uses a relative path to the current file. Another way might be to use `settings.PROJECT_ROOT`, like [here](https://github.com/cfpb/cfgov-refresh/blob/6e44c2f113d15510ea565ca2493fb6f87bea2a80/cfgov/cfgov/test.py#L67).